resources/schema.gql

```diff
@@ -233,6 +237,10 @@ type Dataset {
 	"""
 	metadata: DatasetMetadata!
 	"""
+	Access to the flow configurations of this dataset
+	"""
+	flowConfigs: DatasetFlowConfigs!
```

Comment:
Would it be better to add some extra nesting here to have a whole group dedicated to flows?

I imagine we will have more than just config management, but also accessing history of flows, triggering etc. so I would prefer all of those to be grouped together instead of adding more stuff to `Dataset` node.

Reply:
If you agree - let's make sure that mutations also reside in such a group.

Reply:
Yes, it makes sense, it will likely grow in this direction.